testcontainers/mssql.py

```diff
@@ -0,0 +1,42 @@
+from os import environ
+
+from testcontainers.core.generic import DbContainer
+
+
+class SqlServerContainer(DbContainer):
+    SQLSERVER_PASSWORD = environ.get("SQLSERVER_PASSWORD", "1Secure*Password1")
+    SQLSERVER_USER = "SA"
+    SQLSERVER_DBNAME = "tempdb"
+    SQLSERVER_DRIVER = "ODBC Driver 17 for SQL Server"
+
+    def __init__(self, image="mcr.microsoft.com/mssql/server:2019-latest", **kwargs):
+        super(SqlServerContainer, self).__init__(image)
+        self.port_to_expose = 1433
```

Comment:
Can we add `port_to_expose` to the arguments in case someone wants to spin up two of these?

Reply:
Done!